G/GMT/build_tarballs.jl

```diff
@@ -0,0 +1,89 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "GMT"
+version = v"6.4.0"
+GSHHG_VERSION="2.3.7"
+DCW_VERSION="2.1.1"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/GenericMappingTools/gmt", 
+    "19413d486888ebdc3c01aeed3707bcc88eb727df"),
+
+    ArchiveSource("https://github.com/GenericMappingTools/gshhg-gmt/releases/download/$GSHHG_VERSION/gshhg-gmt-$GSHHG_VERSION.tar.gz",
+        "9bb1a956fca0718c083bef842e625797535a00ce81f175df08b042c2a92cfe7f"),
+
+    ArchiveSource("https://github.com/GenericMappingTools/dcw-gmt/releases/download/$DCW_VERSION/dcw-gmt-$DCW_VERSION.tar.gz",
+        "d4e208dca88fbf42cba1bb440fbd96ea2f932185c86001f327ed0c7b65d27af1")
+]
+
+# Bash recipe for building across all platforms
+script = """
+GSSHG_VERSION="$(GSHHG_VERSION)"
+GSSHG="gshhg-gmt-$(GSHHG_VERSION)"
+DCW="dcw-gmt-$(DCW_VERSION)"
+""" * raw"""
+
+
+cd $WORKSPACE/srcdir
+cd gmt
+mkdir build
+cd build/
+
+cmake -DCMAKE_INSTALL_PREFIX=${prefix} \
+    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DHAVE_QSORT_R_GLIBC=False \
+    -DHAVE___BUILTIN_BSWAP16=False \
+    -DHAVE___BUILTIN_BSWAP32=False \
+    -DHAVE___BUILTIN_BSWAP64=False \
+    -DGMT_ENABLE_OPENMP=True \
+    -DGSHHG_PATH=/workspace/srcdir/${GSSHG} \
+    -DGSHHG_VERSION=${GSHHG_VERSION_numeric} \
+    -DDCW_PATH=/workspace/srcdir/${DCW} \
+    .. 
+make -j${nproc} 
+make install 
+
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
+    Platform("x86_64", "linux"; libc = "glibc"),
+    Platform("aarch64", "linux"; libc = "glibc"),
+    Platform("x86_64", "macos"; ),
+    Platform("aarch64", "macos"; ),
+    Platform("x86_64", "windows"; ),
+    Platform("i686", "windows"; ),
+]
+
+
+# The products that we will ensure are always built
+products = [
+    LibraryProduct("libpostscriptlight", :libpostscriptlight),
```

Comment:
Same here. When products have different basenames on different platforms you have to list them. Please read the documentation: https://docs.binarybuilder.org/stable/reference/#BinaryBuilderBase.LibraryProduct